app/views/application/_pickup_posts.html.haml

```diff
@@ -0,0 +1,5 @@
+- if pickup_posts.count >= 1
```

Comment:
Can you use `exists?` instead of `count >= 1`?
The former is optimized for this purpose. 
``` irb
[1] pry(main)> Post.exists?
  Post Exists (1.6ms)  SELECT  1 AS one FROM "posts" LIMIT 1
=> true
[2] pry(main)> Post.count
   (43.1ms)  SELECT COUNT(*) FROM "posts"
=> 10234
```